internal/zinc-ivy-integration/src/main/scala/sbt/internal/inc/ZincComponentCompiler.scala

```diff
@@ -42,9 +42,12 @@ private[sbt] object ZincComponentCompiler {
     def compilerBridgeId(scalaVersion: String) = {
       // Defaults to bridge for 2.12 for Scala versions bigger than 2.12.x
       scalaVersion match {
-        case sc if (sc startsWith "2.10.") => "compiler-bridge_2.10"
-        case sc if (sc startsWith "2.11.") => "compiler-bridge_2.11"
-        case _                             => "compiler-bridge_2.12"
+        case sc if (sc startsWith "2.10.")     => "compiler-bridge_2.10"
+        case sc if (sc startsWith "2.11.")     => "compiler-bridge_2.11"
+        case sc if (sc startsWith "2.12.")     => "compiler-bridge_2.12"
+        case sc if (sc startsWith "2.13.0-M")  => "compiler-bridge_2.13.0-M2"
```

Comment:
https://github.com/sbt/sbt/issues/3771

Should we add follows before this line? 

```scala
case sc if (sc startsWith "2.13.0-M1")  => "compiler-bridge_2.12"
```

Reply:
I'll accept a PR, but I am sort of ok if it didn't work with old M1.